data/tooltips.txt

```diff
@@ -75,10 +75,10 @@ tip "cargo scan:"
 	`Lets you scan your target ship's cargo from up to this distance away.`
 
 tip "cargo scan power:"
-	`The range of this cargo scanner is proportional to the square root of this value.`
+	`The range of this cargo scanner is proportional to the square root of this value. At increasing distances, the scanner's speed will naturally decline.`
```

Comment:
Although distance to target does decrease scan speed, I think factors that effect speed should only be mentioned in the speed tooltip.
```suggestion
	`The range of this cargo scanner is proportional to the square root of this value.`
```